students/itsabhi/Lesson3/mailroom.py

```diff
@@ -0,0 +1,73 @@
+#! /usr/bin/env python3
+
+
+def sending_thanks(d):
+    donor_hist = d
+    unique_donors = []
+    for entry in donor_hist:
+        name = entry[0]
+        if name not in unique_donors:
+            unique_donors.append(name)
+
+    donor = input("Type the Full Name of the donor here: ")
+    if donor == "list":
+        print(unique_donors)
+        donor = input("Type the Full Name of the donor here: ")
+    if donor not in unique_donors:
+        donor_hist.append([donor])
+        amount = input("Enter Donation value: ")
+        donor_hist[-1].append([amount])
+#        print(donor_hist)
```

Comment:
Remember to convert the `amount` to a numerical type so you can use it in your report calculations